From 5daa31fc86b59e2dbbad43f4b57caa93fd622454 Mon Sep 17 00:00:00 2001
From: vrochalo <vinicius.rochalobo@windriver.com>
Date: Fri, 7 Mar 2025 11:42:17 -0300
Subject: [PATCH] Add service role to neutron policy

Temporary openstack-helm patch bringing the code fix [1] for the reported launchpad [2].
Since OpenStack 2023.2, Neutron user needs to be migrated to service accounts.
 
[1] https://github.com/openstack/openstack-helm/commit/5708319cd8acf4edbe31d8416da52b89e8a97fd5
[2] https://bugs.launchpad.net/openstack-helm/+bug/2078002

Signed-off-by: vrochalo <vinicius.rochalobo@windriver.com>
---
 neutron/values.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/neutron/values.yaml b/neutron/values.yaml
index b7a5560d..8a40e41b 100644
--- a/neutron/values.yaml
+++ b/neutron/values.yaml
@@ -2448,7 +2448,7 @@ endpoints:
         user_domain_name: default
         project_domain_name: default
       neutron:
-        role: admin
+        role: admin,service
         region_name: RegionOne
         username: neutron
         password: password
-- 
2.34.1

